### PR TITLE
chore: add app insights sdk to component explorer website

### PR DESCRIPTION
**Pull request by awentzel**
# Description
This additional enables application telemetry integration using [Azure Insights](https://github.com/Microsoft/ApplicationInsights-js) and monitors your backend services and components after you deploy them to help you discover and rapidly diagnose performance and other issues.

## Motivation & context
All of our websites require telemetry and integration to Azure so we can properly monitor the health of our applications and proactively act when necessary.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by janechu** [comment]
How have these changes been tested?

**Comment by awentzel** [comment]
@janechu - these have been tested through Azure which when the app is running collects application data and enables a mechanism to view and analyze.

**Comment by janechu** [comment]
Should we have separate instrumentation keys for development and production so that we are only gathering data from the live site? See: https://docs.microsoft.com/en-us/azure/azure-monitor/app/separate-resources

**Comment by awentzel** [comment]
@janechu sure that can be added in a future PR.  Once this goes in there will be further testing and configuration necessary.  That would be a good point to add it.  Additionally, we'll want to break this out in a class of its own so it's easily replicated across websites source code.

**Comment by janechu** [comment]
If we already know we want to break this out now, I would suggest that we include that work in this PR. Is there a reason we can't do that yet?

**Comment by awentzel** [comment]
@janechu because I don't have the bandwidth to do the other stuff at this time which will also take further research and investigation.

**Comment by nicholasrice** [comment]
I really don't think we should be checking this token into source control.

File: packages/fast-component-explorer/app/index.tsx [hunk 1]
```diff
@@ -11,10 +14,30 @@ const root: HTMLElement = document.createElement("div");
...
-const DragDropApp: any = DragDropContext(HTML5Backend)(App);
+/**
+ * Create Application Insights SDK for Telemetry
+ * https://docs.microsoft.com/en-us/azure/azure-monitor/learn/nodejs-quick-start
+ */
+const browserHistory = createBrowserHistory({ basename: "" });
+var reactPlugin = new ReactPlugin();
+var appInsights = new ApplicationInsights({
+    config: {
+        instrumentationKey: "ecda4203-b38b-42ad-b181-f0f7f2b1d8fa",
```
Comment (nicholasrice) [inline packages/fast-component-explorer/app/index.tsx]: I understand this isn't a security threat, leaving this in source for the world to see opens the opportunity for this to get stolen and misused.

We need to tokenize this as part of a build step so private information stays private.

**Comment by janechu** [comment]
@nicholasrice I agree, it leaves us open to data being passed that does not originate from the relevant sites.

I would like to see:
- Tokens used in place of generated keys
- Tokens for production and staging
- Factoring out the logic for use in both the explore site and the doc site if this is expected to happen in the future

File: packages/fast-component-explorer/app/index.tsx (hunk 1 above)
Comment (awentzel) [reply packages/fast-component-explorer/app/index.tsx]: @nicholasrice Please read https://devblogs.microsoft.com/premier-developer/alternative-way-to-protect-your-application-insights-instrumentation-key-in-javascript/ and then let me know how you'd like to proceed.

**Comment by awentzel** [comment]
@janechu - Could you comment as part of the review so they stay threaded.  For feature requests, your ideas are great but you should file an issue to be prioritized and allocated as availability allows.

File: packages/fast-component-explorer/app/index.tsx (hunk 1 above)
Comment (janechu) [reply packages/fast-component-explorer/app/index.tsx]: *Added this to the original comment thread*

@nicholasrice I agree, it leaves us open to data being passed that does not originate from the relevant sites.

I would like to see:

Tokens used in place of generated keys
Tokens for production and staging
Factoring out the logic for use in both the explore site and the doc site if this is expected to happen in the future

Comment (nicholasrice) [reply packages/fast-component-explorer/app/index.tsx]: My understanding is that, with this implementation, anyone that comes across this key could pollute our data - is that correct?

Comment (nicholasrice) [reply packages/fast-component-explorer/app/index.tsx]: My understanding is that, with this implementation, anyone that comes across this key could pollute our data - is that correct?

**Pull request closed without merging**